cluster/gce/upgrade.sh

```diff
@@ -392,20 +406,20 @@ function do-node-upgrade() {
       return ${set_instance_template_rc}
     fi
     instances=()
-    instances+=($(gcloud compute instance-groups managed list-instances "${group}" \
+    while IFS='' read -r line; do instances+=("$line"); done < <(gcloud compute instance-groups managed list-instances "${group}" \
         --format='value(instance)' \
         --project="${PROJECT}" \
-        --zone="${ZONE}" 2>&1)) && list_instances_rc=$? || list_instances_rc=$?
+        --zone="${ZONE}" 2>&1) && list_instances_rc=$? || list_instances_rc=$?
     if [[ "${list_instances_rc}" != 0 ]]; then
       echo "== FAILED to list instances in group ${group} =="
-      echo "${instances}"
+      echo "${instances[@]}"
       return ${list_instances_rc}
     fi
 
     process_count_left=${node_upgrade_parallelism}
     pids=()
     ret_code_sum=0  # Should stay 0 in the loop iff all parallel node upgrades succeed.
-    for instance in ${instances[@]}; do
+    for instance in "${instances[@]}"; do
```

Comment:
are you _sure_ this works exactly the same as before? I think we actually want globbing/splitting here